assets/pages/new-contributions-landing/components/ContributionFormFields.jsx

```diff
@@ -100,6 +111,11 @@ function FormFields(props: PropTypes) {
         disabled={isSignedIn}
       />
       <Signout isSignedIn />
+      <MustSignIn
```

Comment:
Seeing as all these props are read directly from the state, I'd rather we just read them from the state in `MustSignIn` rather than passing them through here.

Reply:
we want @tomrf1 to re-use this component, so we'll pass them in